lxd/db/cluster/entities.go

```diff
@@ -6,24 +6,24 @@ import (
 
 // Numeric type codes identifying different kind of entities.
 const (
-	TypeContainer             = 0
-	TypeImage                 = 1
-	TypeProfile               = 2
-	TypeProject               = 3
-	TypeCertificate           = 4
-	TypeInstance              = 5
-	TypeInstanceBackup        = 6
-	TypeInstanceSnapshot      = 7
-	TypeNetwork               = 8
-	TypeNetworkACL            = 9
-	TypeNode                  = 10
-	TypeOperation             = 11
-	TypeStoragePool           = 12
-	TypeStorageVolume         = 13
-	TypeStorageVolumeBackup   = 14
-	TypeStorageVolumeSnapshot = 15
-	TypeWarning               = 16
-	TypeClusterGroup          = 17
```

Comment:
I actually prefer the explicit nature of the previous version because it means that if for any reason we remove a type in the future it wont reshuffle the IDs implicitly.

Reply:
I didn't think the actual IDs would be important since we should always be using the constants but I have dropped the commit and added `TypeStorageBucket = 18`

Reply:
thanks!